recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
 
 build:
   skip: true  # [win and vc<14]
```

Comment:
To avoid these issues in the future
```suggestion
  skip: true  # [win and vc<14]
  error_overlinking: true
```

Reply:
@isuruf, thanks, I didn't know about that option.